setup.cfg

```diff
@@ -35,6 +35,7 @@ install_requires =
     gwcs>=0.16.1
     jsonschema>=3.0.2
     numpy>=1.17
+    numba>=0.50.0
```

Comment:
Now that we're using C extensions instead of numba, can this be removed?

Reply:
yes I forgot I put that in